package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "anomaly-detection-dashboards",
-  "version": "2.11.0.0",
+  "version": "2.11.1.0",
```

Comment:
Can you also update `plugin_version` on line 7? This is to prevent builds from breaking when the zip is created.